nav2_bringup/launch/bt_navigator.xml

```diff
@@ -21,7 +21,9 @@
       <RateController hz="1.0">
         <ComputePathToPose goal="${goal}"/>
       </RateController>
-      <FollowPath path="${path}"/>
+      <SequenceStar name="RememberFollowPathResult">
```

Comment:
Don't fully understand why we need to wrap a `SequenceStar` around `FollowPath`.
We could add something about it on the description above.

Reply:
@orduno I added this to ensure that FollowPath would only ever be called once. It was clear to me whether there could be a condition where the sequence node would invoke FollowPath again, which we don't want.